test/demo/start

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import sys
+
+from drenv import kubectl
+
+if len(sys.argv) != 2:
+    print(f"Usage: {sys.argv[0]} cluster")
+    sys.exit(1)
+
+cluster = sys.argv[1]
+
+print("Deploying demo")
+kubectl.apply("--filename", "demo/deployment.yaml", context=cluster)
+
+print("Creating the service")
+kubectl.apply("--filename", "demo/service.yaml", context=cluster)
+
+print("Waiting until deployment is rolled out")
+kubectl.rollout(
+    "status",
+    "deploy/demo-deployment",
+    "--timeout=360s",
```

Comment:
Deployment is very quick, we don't need 360 seconds. Maybe use 120s?

Same for other timeouts in this file.